composer.json

```diff
@@ -20,7 +20,7 @@
     "require-dev": {
         "symfony/yaml": "~2.1",
         "symfony/dependency-injection": "~2.0",
-        "phpspec/prophecy": "~1.1.2"
+        "phpunit/phpunit": "~4.5"
```

Comment:
But it does depend on the 4.5 version for the prophecy support (and I am not sure what version travis uses).

Otherwise it is a question of if we want to use the native prophecy support at this point. I vote for yes.